src/priorityQueue.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+export default class priorityQueue {
+  constructor(){
```

Comment:
Add a space before the `{` - shows greater consistency with the syntax guidelines you've been following and makes your code more readable.